src/core/path_coord.cpp

```diff
@@ -467,8 +467,9 @@ SplitPathCoord SplitPathCoord::at(
 			
 			if (prev_coord.index == first.index)
 			{
+				Q_ASSERT(split.param >= first.param);
+				Q_ASSERT(1.0 >= first.param);
 				auto p = (split.param - first.param) / (1.0f - first.param);
```

Comment:
This branch is suspicious: 
`first` is not `prev_coord`, even when the point to the same `index`.